spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/TestSparkPartitionSerialization.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.spark;
+
+import java.io.IOException;
+import java.util.Map;
+import org.apache.iceberg.KryoHelpers;
+import org.apache.iceberg.TestHelpers;
+import org.apache.iceberg.relocated.com.google.common.collect.ImmutableMap;
+import org.apache.iceberg.spark.SparkTableUtil.SparkPartition;
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+
+public class TestSparkPartitionSerialization {
```

Comment:
I think we can just use `TestSparkTableUtil` as test class name

Reply:
Yeah that seems reasonable.

Reply:
Updated it to be `TestSparkTableUtil`. In Spark 2.4, there already is a `TestSparkTableUtil` so I just added it into that class (those tests are somewhere more specific in Spark 3+).